### ISSUE TITLE
Show repositories committed to, not just repositories in account

### ISSUE THREAD

**Issue by tedivm** [resume/resume.github.com#197]
The bulk of my commits are in repositories that are in an organization, which really skews the results.

**Comment by kernc** [comment]
> Contributes to ...

https://api.github.com/search/issues?q=type:pr+user:username

**Comment by maglnet** [comment]
The link by @kernc only recognizes own repositories not those by others.
I think this would be the correct search query:

https://api.github.com/search/issues?q=type:pr+author:username

**Comment by zwass** [comment]
This seems super important to me, especially given that a user's most meaningful contributions are likely to be to a widely used open source project, rather than a pet project in a personal repo.

**Comment by michaelachrisco** [comment]
Agreed. I am thinking `Popular Repositories` can become `Popular Personal Repositories` and a new field can be added called `Popular Open Source Repositories` more neat the top.

**Comment by Rolinh** [comment]
@kernc @maglnet 
This would not be sufficient. You can be part of an organization with the commit bit for the organization's repositories.

**Comment by tedivm** [comment]
This project hasn't had in a commit in 18 months- if you guys want to see changes you're probably going to need to fork it and make them yourself.

**Comment by u2** [comment]
#216 
Yes, it's a very important feature, even more than the owned repos. Is there someone that would like to contribute for this?

**Comment by u2** [comment]
I have added contributions section, but this is imperfect. You can see it http://resume.github.io/?u2 .

**Comment by greenlaw110** [comment]
The contributions section doesn't report my heavy commits in some organizations:

It reports:

![image](https://cloud.githubusercontent.com/assets/216930/15702437/5ce2e2fa-2823-11e6-8cf1-e913ce0d3387.png)

While my real commits are here:

![image](https://cloud.githubusercontent.com/assets/216930/15702469/7fc49cdc-2823-11e6-811a-55ef65dd3cc0.png)

and here:

![image](https://cloud.githubusercontent.com/assets/216930/15702480/9283e378-2823-11e6-9001-db03d13a4b4c.png)
